bindings/python/malamute/_malamute_ctypes.py

```diff
@@ -7,7 +7,7 @@
 import os, sys
 from ctypes import *
 from ctypes.util import find_library
-import czmq
+import czmq.git
```

Comment:
It was using the directory name before. The problem is that, as far as I can see, there's no other way to get the project name other than from the directory name at that stage